features/invoker-commands.yml

```
@@ -0,0 +1,12 @@
+name: Invoker Commands
+description: Adding commandfor and command attributes to elements would allow to assign interactions in a more accessible and declarative way, while reducing bugs and simplifying the amount of JavaScript pages are required to ship for interactivity.
```

```suggestion
description:  The `command` HTML attribute declaratively creates an interaction on a `<button>` element without JavaScript. The interaction target is specified with the `commandFor` HTML attribute on the same `<button>` element. 
```

This focuses more on how someone can use it, rather than the why.